schematic/models/validate_attribute.py

```diff
@@ -583,8 +596,13 @@ def list_validation(
         if list_robustness == "strict":
             # This will capture any if an entry is not formatted properly. Only for strict lists
             for i, list_string in enumerate(manifest_col):
-                if not re.fullmatch(csv_re, list_string):
+                list_error = None
+                if not isinstance(list_string, str):
+                    list_error = "not_a_string"
+                elif not re.fullmatch(csv_re, list_string):
                     list_error = "not_comma_delimited"
+
+                if list_error:
                     vr_errors, vr_warnings = GenerateError.generate_list_error(
```

Comment:
You might consider breaking some of the nested parts into its own method/function